cli_reference/developer_cli_odo/odo-release-notes.adoc

```diff
@@ -8,48 +8,22 @@ toc::[]
 
-* Dynamic registry support. Now you can configure your own registries with the following commands:
-+
-[source,terminal]
-----
-# Add a registry to the registry list
-odo registry add <registry name> <registry URL>
-
-# List a registry in the registry list
-odo registry list
-
-# Delete a registry from the registry list
-odo registry delete <registry name>
-
-# Update a registry in the registry list
-odo registry update <registry name> <registry URL>
-
-# List a component with a corresponding registry
-odo catalog list components
-
-# Create a component that is hosted by a specific registry
-odo create <component type> --registry <registry name>
-----
-
-* The `--starter` flag is added to `odo create`. Run `odo create nodejs --starter <project-name>` to download the source code of a project specified in the devfile. If no project name is specified, `odo` downloads the first one.
-
-* The `--context` flag is added to `odo push`. With `--context`, you can trigger `odo push` from outside the source code directory. Run `odo push --devfile <path to the devfile> --context <directory with your component>` to specify the directory of your component.
+* The default deployment mechanism in `{odo-title}` is the devfile deployment now.
 
-* Performance improvement for `odo catalog list components` when using the devfiles.
+* Added documentation about converting S2I to devfile. See, xref:../../cli_reference/developer_cli_odo/using-devfiles-in-odo.adoc#converting-an-s2i-component-into-a-devfile-component_creating-applications-by-using-devfiles[Converting an S2I component into a devfile component].
 
-* The `--now` flag is added for `odo url delete` when using the devfiles.
+* Added commands to display JSON output for devfile components. 
 
-* `odo url delete --now` now works with the devfiles.
-* The `--debug` flag now works with the devfiles.
-
-* Added machine-readable output for listing Operator-backed services. Run `odo catalog list services -o json` to display information about Operators and services in JSON format.
+* `odo debug` is is no longer a Technology Preview feature and is now fully supported.
+To learn more, see xref:../../cli_reference/developer_cli_odo/debugging-applications-in-odo.html#debugging-an-application_debugging-applications-in-odo[Debugging applications in odo].
 
-* Added machine-readable output for debugging. Run `odo debug info -o json` to display the debugging information in JSON format.
+* `{odo-title}` integration wit Operators support. You can now create services from Operators with `{odo-title}` is no longer a Technology Preview feature and is now fully supported. To learn more, see xref:../../cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-operators.adoc[Creating instances of services managed by Operators]. 
```

Comment:
s/wit/with

Reply:
This doesn't sound right : ` You can now create services from Operators with odo is no longer a Technology Preview feature and is now fully supported. ` 

Something's missing as a punctuation maybe?